### PR TITLE
[SHELL32] *.rc 640x480-ify IDD_FOLDER_OPTIONS_*

### PR DESCRIPTION
**Pull request by JoachimHenze**
This makes the 3 dlgs fit the screen in y for 640x480 desktop resolution like their MS role-models.

Beside that also fixes:
* the wrong vertical position of IDC_FOLDER_OPTIONS_TASKICON and wrong horizontal position of ICON IDI_SHELL_PROGRAMS_FOLDER2 for all translations
* fix some accelerator collisions for de-DE and place many FIXMEs for other translations that still have some
* improve the distance on the lower end (height) of the 3 groupboxes to look much better than before in IDD_FOLDER_OPTIONS_GENERAL for all langs
* bg-BC.rc in 1st dlg IDD_FOLDER_OPTIONS_GENERAL the text of the radiobuttons was historically shifted to the left (as part of an ancient workaround for the too long strings) and as a consquence the text was at the same area where the icons are. Fixed now.
* en-US: text truncation for IDC_FOLDER_OPTIONS_DOUBLECLICK string (affected also many other langs)
* es-ES: fix text truncation on button IDC_FOLDER_OPTIONS_RESTORE
* id-ID: BS_MULTILINE in IDC_FOLDER_OPTIONS_ULBROWSER and IDC_FOLDER_OPTIONS_ULPOINT. Added missing spacing between IDC_VIEW_APPLY_TO_ALL and IDC_VIEW_RESET_ALL buttons
* it-IT.rc fix text truncation in IDC_FOLDER_OPTIONS_RESTORE
* ro-RO.rc fix text truncation in IDC_FOLDER_OPTIONS_RESTORE
* uk-UA.rc lacked the ICON IDI_SHELL_PROGRAMS_FOLDER2 entirely (as a consequence of a historic workaround)
* fix text truncation for many languages for the radiobuttons in IDD_FOLDER_OPTIONS_GENERAL (1st dlg)
* fix text truncation for all languages for the "Opens with:" text in IDD_FOLDER_OPTIONS_FILETYPES (3rd dlg)
* strip unneeded dummy texts for IDC_FILETYPES_APPNAME
* take over "Free Space" instead of "Size available" for IDS_SHV_COLUMN_DISK_AVAILABLE which was done for en-US.rc some years ago, but was forgotten to do as well for other languages back then
* ...
* actually this PR includes **hundreds of fixes**, too many to name all of them, therefore I decided to stop listing them here and instead let some after-videos speak for themselves

## Purpose

de-DE after video (final state):
[video-2023-12-17T02-42-13-881875000Z.webm](https://github.com/reactos/reactos/assets/33393466/6cb97d5a-e9db-4189-a0d4-e6c728677dff)

en-US after video (final state):
[video-2023-12-17T02-37-57-366250000Z.webm](https://github.com/reactos/reactos/assets/33393466/53c5314c-1fbb-4f8a-9e5f-c74166ea546f)

fr-FR after video (final state) (an example for fancy BS_MULTILINE within IDD_FOLDER_OPTIONS_GENERAL, imho shortening the strings instead would look superior):
[video-2023-12-17T02-47-10-803750000Z.webm](https://github.com/reactos/reactos/assets/33393466/368e0ffd-7e58-4208-880a-0ec562439bd7)

id-ID after video (final state) (also got fixes for BS_MULTILINE in IDC_FOLDER_OPTIONS_ULBROWSER and IDC_FOLDER_OPTIONS_ULPOINT. Added missing spacing between IDC_VIEW_APPLY_TO_ALL and IDC_VIEW_RESET_ALL):
[video-2023-12-17T03-07-46-116250000Z.webm](https://github.com/reactos/reactos/assets/33393466/6fcaeb6c-77b4-4e43-8d69-208ced69a204)

es-ES after video (final state) (as an example which is one of the most poor looking versions, the 3 dialogs dimensions have been artificially increased in x by other translators to have lots of space available. Will make future ros development harder than necessary. Also notice the ugly looking button in the last tab due to the whole holy bible being written to that. Since I did not want to enter political discussions, I decided to stick with the increased dimensions):
[video-2023-12-17T02-52-07-772500000Z.webm](https://github.com/reactos/reactos/assets/33393466/03be9a77-bfd0-4deb-82b0-f2da1f773c90)

pl-PL after video (final state):
[video-2023-12-16T23-48-24-647500000Z.webm](https://github.com/reactos/reactos/assets/33393466/c92dbb9f-6768-4018-8e1e-377a3cc5c6b5)

**Ultimate ResHacker preview video for all rc IDD_FOLDER_OPTIONS_GENERAL** (all rcs where the fonts are installed)
[video-2023-12-17T03-17-35-256875000Z.webm](https://github.com/reactos/reactos/assets/33393466/c13e926a-837e-4c41-9c47-7d741ddc78c6)

**Ultimate ResHacker preview video for all rc IDD_FOLDER_OPTIONS_VIEW** (all rcs where the fonts are installed)
[video-2023-12-17T03-25-49-928750000Z.webm](https://github.com/reactos/reactos/assets/33393466/173da6e9-4ca8-41b9-8e5d-e15f4205f547)

**Ultimate ResHacker preview video for all rc IDD_FOLDER_OPTIONS_FILETYPES** (all rcs where the fonts are installed)
[video-2023-12-17T03-31-55-053750000Z.webm](https://github.com/reactos/reactos/assets/33393466/80944d00-f5ce-4a18-b594-3c257c608fe6)

JIRA issue: [CORE-1762](https://jira.reactos.org/browse/CORE-1762) is the closest match, although it doesn't deal with "just" the IDD_FOLDER_OPTIONS.

## TODO

- [ ] **in a different PR** find out why the treeview IDC_VIEW_TREEVIEW is scrolled topmost in en-US, but is not in de-DE. That looks suboptimal but is not caused by my PR, was like this before already.

### CODE REVIEW

**Comment by whindsaks** [comment]
I don't see this clashing with [6122](https://github.com/reactos/reactos/pull/6122) but keep it in mind anyway.

I was planning to do something about the icon as bmp issue on the types tab but I guess I'll have to wait on both of these now so we don't conflict.

**Comment by JoachimHenze** [comment]
impressive. hi-ID can indeed be satisfied with 48 width already for "के साथ खुलता है:" == "Opens with". That hindi glpyhs are very narrow. Nice.
image

**Review by katahiromz — APPROVED** [review]

File: dll/win32/shell32/lang/tr-TR.rc [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * PURPOSE:     Turkish resource file
  * TRANSLATORS: Copyright 2006 Fatih Aşıcı <fasici@linux-sevenler.org>
  *              Copyright 2012 Arda Tanrıkulu (ardatan) <ardatanrikulu@gmail.com>
- *              Copyright 2015-2016, 2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
+ *              Copyright 2015-2020 Erdem Ersoy (eersoy93)
```
Comment (binarymaster) [inline dll/win32/shell32/lang/tr-TR.rc:7]: Please do not remove email even if it's outdated information. (I'm fine with correcting the year, but please also make it a separate commit in master)

**Review by binarymaster — APPROVED** [review]
Spacing between the tabs and the first group control (on all pages) seems a bit large to me – but I didn't checked whether it's introduced by this PR. Otherwise, LGTM.

image

File: dll/win32/shell32/lang/tr-TR.rc (hunk 1 above)
Comment (JoachimHenze) [reply dll/win32/shell32/lang/tr-TR.rc:7]: I do see no reason why this single-line-comment-change should be a separate commit. You do mix such header changes into everybody elses PRs all the time. You are absolutely applying double-standards to how you work and to how you do expect anybody else to work.

**Comment by JoachimHenze** [comment]
> Spacing between the tabs and the first group control (on all pages) seems a bit large to me – but I didn't checked whether it's introduced by this PR. Otherwise, LGTM.
> 
> image

I did **not** touch or change this aspect within my PR. The same distance is currently used in all 3 dialogs, yes.

File: dll/win32/shell32/lang/tr-TR.rc (hunk 1 above)
Comment (learn-more) [reply dll/win32/shell32/lang/tr-TR.rc:7]: It's not about a separate commit, it is about you removing the email address.

**Review by learn-more — CHANGES_REQUESTED** [review]
Please restore the email address

**Review by binarymaster — APPROVED** [review]

**Review by DarkFire01 — APPROVED** [review]

**Comment by HBelusca** [comment]
@JoachimHenze your JIRA report you're searching for is: https://jira.reactos.org/browse/CORE-1762 (please update the PR description and commit log, if applicable)

**Review by HBelusca — APPROVED** [review]

**Comment by JoachimHenze** [comment]
> @JoachimHenze your JIRA report you're searching for is: https://jira.reactos.org/browse/CORE-1762 (please update the PR description and commit log, if applicable)

Thank you for the search. The one you found isn't exactly the one I was looking for. There was one that I resolved as WontFix personally myself historically which was explicitly about the IDD_FOLDER_OPTIONS_* . But yours is better than none at all, so I will link to that instead.
